scripts/manual/API_REFERENCE_WORKFLOW.md

```
@@ -1,129 +1,207 @@
+   - Example: `/api/v1/organizations/{id}` → `api_v1_organizations_{id}`
+   - Languages: `javascript` (`.js`), `python` (`.py`), `go` (`.go`), `java` (`.java`)
+5. Run:
 
 ```bash
-pnpm run generate-search-index
+pnpm run bundle:apis
+pnpm run validate-api-split
 ```
 
-This runs the same `extractApiRecords` logic and prints URL checks in the terminal. It does **not** write files or update Algolia; use it after OpenAPI or `generateUrlFragment()` changes to confirm fragments still match Scalar.
+6. Preview with `pnpm start` and open `/apis`, `/agentkit/apis`, or `/saaskit/apis`.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 6; Style: 1/1/1

(MD029, ol-prefix)